### PR TITLE
Use es6 includes for d3 and momentjs

### PR DESCRIPTION
**Pull request by Lukewh**
Supercedes https://github.com/canonical-websites/snapcraft.io/pull/944

# Done

- Replaced minified built versions of d3, topojson and moment.js with es6 imports
- Added entire `dist` and `modules` folders to `.gitignore`

# TODO

- Implement d3 version of metrics graphs to remove billboard dependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build": "yarn run build-js && yarn run build-css",
     "build-css": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' && postcss --use cssnano --dir static/minified 'static/css/**/*.css'",
     "build-js": "yarn run copy-3rd-party-js && yarn run build-js-transpile",
-    "copy-3rd-party-js": "cp node_modules/d3/build/d3.min.js static/js/modules && cp node_modules/d3-geo/build/d3-geo.min.js static/js/modules && cp node_modules/topojson-client/dist/topojson-client.min.js static/js/modules && cp node_modules/billboard.js/dist/billboard.min.js static/js/modules && cp node_modules/moment/min/moment.min.js static/js/modules && cp node_modules/clipboard/dist/clipboard.min.js static/js/modules && cp node_modules/raven-js/dist/raven.min.js static/js/modules",
+    "copy-3rd-party-js": "cp node_modules/d3/build/d3.min.js static/js/modules && cp node_modules/billboard.js/dist/billboard.min.js static/js/modules && cp node_modules/moment/min/moment.min.js static/js/modules && cp node_modules/clipboard/dist/clipboard.min.js static/js/modules && cp node_modules/raven-js/dist/raven.min.js static/js/modules",
```
Comment (bartaz) [inline package.json]: Is `moment.min.js` still needed as separate script?

File: static/js/publisher/metrics/graphs/activeDevices.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
-/* globals bb, moment */
+/* globals bb */
```
Comment (bartaz) [inline static/js/publisher/metrics/graphs/activeDevices.js]: Can't `bb` be imported as well?

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
...
     "build-js-transpile": "rollup -c",
```
Comment (bartaz) [inline package.json]: Not directly related to this PR, but just noticed we don't minify our JS at all (just bundle). Now, with d3 and moment bundled together `public.js` just got pretty large. We should add minify step at some point.

File: static/js/publisher/metrics/graphs/activeDevices.js (hunk 2 above)
Comment (Lukewh) [reply static/js/publisher/metrics/graphs/activeDevices.js]: unfortunately not! That's one of the reasons I want to rewrite it!

File: package.json (hunk 3 above)
Comment (Lukewh) [reply package.json]: I've added an uglify step to the rollup.config.js's

File: package.json (hunk 1 above)
Comment (Lukewh) [reply package.json]: Removed! Thanks @bartaz

**Comment by bartaz** [comment]
Awesome, thanks @Lukewh!

**Pull request merged**